test/extended/networking.sh

```diff
@@ -212,10 +212,10 @@ function run-extended-tests() {
 
   if [[ -n "${dlv_debug}" ]]; then
     # run tests using delve debugger
-    local test_cmd="dlv exec ${TEST_BINARY} -- ${test_args}"
+    local test_cmd="dlv exec extended.test -- ${test_args}"
```

Comment:
Actually nevermind, if we're not using it as an executable I'll continue to pass the path.